sqlite3/lib/src/api/exception.dart

```diff
@@ -21,14 +21,24 @@ class SqliteException implements Exception {
   /// code, providing some idea of the cause of the failure.
   int get resultCode => extendedResultCode & 0xFF;
 
-  SqliteException(this.extendedResultCode, this.message, [this.explanation]);
+  /// Last SQL command executed
+  String sqlCommand;
+
+  SqliteException(this.extendedResultCode, this.message, this.sqlCommand,
+      [this.explanation]);
```

Comment:
Changing the constructor like this is a breaking change. Can we make `sqlCommand` nullable and then use this instead?

```suggestion
  SqliteException(this.extendedResultCode, this.message,
      [this.explanation, this.sqlCommand]);
```